src/reaktoro_pse/core/pyomo_property_writer/property_functions.py

```diff
@@ -29,6 +48,13 @@ def build_scaling_tendency_constraint(rkt_output_object):
 #     build_properties[("saturationIndex", rkt_output_object.property_index)].pyomo_var,
 #     -5,
 # )
+# def build_pOH_constraint(rkt_output_object):
+#     user_output_var = rkt_output_object.pyomo_var
+#     build_properties = rkt_output_object.pyomo_build_options.properties
+#     return (
+#         -build_properties[("speciesActivityLn", "OH-")].pyomo_var / log(10)
+#         == user_output_var
+#     )
```

Comment:
Were you intending on uncommenting, deleting, or reserving for some future time? If keeping it around for future PR, what is holding it up at the moment?